src/apps/metavault-trade/helpers/metavault-trade.mvlp.token-helper.ts

```diff
@@ -4,14 +4,18 @@ import { range } from 'lodash';
 
 import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
 import { buildDollarDisplayItem } from '~app-toolkit/helpers/presentation/display-item.present';
-import { getTokenImg } from '~app-toolkit/helpers/presentation/image.present';
+import { getAppAssetImage } from '~app-toolkit/helpers/presentation/image.present';
 import { ContractType } from '~position/contract.interface';
 import { AppTokenPosition } from '~position/position.interface';
 import { Network } from '~types/network.interface';
 
 import { MetavaultTradeContractFactory } from '../contracts';
 import { METAVAULT_TRADE_DEFINITION } from '../metavault-trade.definition';
 
+const appId = METAVAULT_TRADE_DEFINITION.id;
+const LINK_TOKEN = '0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39'; // Bridged Link Token
+const WHITE_LISTED_LINK_TOKEN = '0xb0897686c545045afc77cf20ec7a532e3120e0f1'; // Native Link Token
```

Comment:
I just added native LINK to our base tokens, feel free to update this and ping me when ready!